oxd-server/pom.xml

```diff
@@ -166,7 +167,25 @@
                     <failOnNoGitDirectory>false</failOnNoGitDirectory>
                 </configuration>
             </plugin>
-
+            <plugin>
```

Comment:
Lets add it into windows-build maven profile, so it is not activated by default.

Reply:
Created `windows-build` maven profile

Reply:
@duttarnab why is it blank? all windows-build related configuration should go inside it. So if on maven package we invoke download and build it must be done within `windows-build` profile. You may wish to read this first http://maven.apache.org/guides/introduction/introduction-to-profiles.html

Reply:
We have added windows-build related configuration in `windows-build` profile.